plexapi/library.py

```diff
@@ -714,3 +726,26 @@ def _loadData(self, data):
 
     def __len__(self):
         return self.size
+
+
+@utils.registerPlexObject
+class Collections(PlexObject):
+
+    TAG = 'Directory'
+    TYPE = 'collection'
+
+    def _loadData(self, data):
+        self.ratingKey = utils.cast(int, data.attrib.get('ratingKey'))
+        self.key = data.attrib.get('key')
+        self.type = data.attrib.get('type')
+        self.title = data.attrib.get('title')
+        self.subtype = data.attrib.get('subtype')
+        self.summary = data.attrib.get('summary')
+        self.index = utils.cast(int, data.attrib.get('index'))
+        self.thumb = data.attrib.get('thumb')
+        self.addedAt = utils.toDatetime(data.attrib.get('addedAt'))
+        self.updatedAt = utils.toDatetime(data.attrib.get('updatedAt'))
+        self.childCount = utils.cast(int, data.attrib.get('childCount'))
+        self.minYear = utils.cast(int, data.attrib.get('minYear'))
+        self.maxYear = utils.cast(int, data.attrib.get('maxYear'))
+        self.children = self.fetchItems(self.key)
```

Comment:
fetchItems does http requests. Change this to a property so it’s only loaded if the user asks for the children.